.github/workflows/release.yaml

```
@@ -3,51 +3,114 @@ name: Release Elasti Helm Chart
+      app_version: ${{ steps.read_chart.outputs.APP_VERSION }}
+      chart_name: ${{ steps.read_chart.outputs.CHART_NAME }}
+      resolver_tag: ${{ steps.read_chart.outputs.RESOLVER_TAG }}
+      operator_tag: ${{ steps.read_chart.outputs.OPERATOR_TAG }}
     steps:
       - name: Checkout main
         uses: actions/checkout@v4
         with:
           fetch-depth: "0"
-    
-      - name: Add yq through apt
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)